admin/fleetcommander/admin.py

```diff
@@ -627,20 +627,37 @@ def websocket_stop(self):
             c.websocket_stop(self.current_session['websockify_pid'])
             del(self.current_session['websockify_pid'])
 
+
 if __name__ == '__main__':
 
-    # Python import
     from argparse import ArgumentParser
-
-    # Fleet commander imports
     from utils import parse_config
 
     parser = ArgumentParser(description='Admin interface server')
     parser.add_argument(
         '--configuration', action='store', metavar='CONFIGFILE', default=None,
         help='Provide a configuration file path for the web service')
+    parser.add_argument(
+        '--host', action='store', metavar='HOST', default=None,
+        help='Listening host for admin application')
+    parser.add_argument(
+        '--port', action='store', metavar='HOST', type=int, default=None,
+        help='Listening port for admin application')
+    parser.add_argument(
+        '--datadir', action='store', metavar='DATADIR', default=None,
+        help='Directory for static admin data')
+    parser.add_argument(
```

Comment:
We should be more specific here, --profiledata and --appdata are probably better option names here. "static admin data" is not very descriptive for a casual user.